daemon/ct.go

```diff
@@ -65,21 +65,23 @@ func (d *Daemon) EnableConntrackGC() {
 		for {
 			sleepTime := time.Duration(GcInterval) * time.Second
 
-			d.endpointsMU.Lock()
+			d.endpointsMU.RLock()
 
 			for k := range d.endpoints {
 				e := d.endpoints[k]
+				e.Mutex.RLock()
 				if e.Consumable == nil {
+					e.Mutex.RUnlock()
 					continue
 				}
-
+				e.Mutex.RUnlock()
```

Comment:
The comment doesn't make sense to me because I don't see `runGC` taking any locks. I assuming accessing `e.ID` and calling `e.LogStatus` is fine without a lock is fine as we are holding `d.endpointsMU` so endpoints can't disappear. I think overall what will help to put a comment to every mutex to describe what the mutex is supposed to protect exactly. Is `e.Mutex` is supposed to protect writes to endpoints but reads are fine?